ALVRClient/Renderer.swift

```diff
@@ -485,7 +487,7 @@ class Renderer {
                                 else {
                                     frameQueue.append(QueuedFrame(imageBuffer: imageBuffer, timestamp: timestamp))
                                 }
-                                if frameQueue.count > 2 {
+                                if frameQueue.count > 1 {
```

Comment:
If we only ever want one queued frame, can we replace the array with just a `QueuedFrame?`

Reply:
Usually you want more than one queued frame. alvr_client_core using the internal MediaCodec decoder uses the setting `max_buffering_frames` to allow for jitter in the decoder latency. This is especially important to account for spikes caused by IDR frames, which might take more than 1/90 seconds to decode.

Reply:
Yeah with further testing, I only occasionally got down to 62ms delay but it seems to be a consistent 72ms for me now